js/map.js

```diff
@@ -54,13 +59,42 @@ function generateMonsters(){
     monsters = [];
     let numMonsters = level+1;
     for(let i=0;i<numMonsters;i++){
-        spawnMonster();
+        spawnMonster(level);
     }
 }
 
-function spawnMonster(){
+function spawnMonster(level){
     // let monsterType = shuffle([Goose, Ant, Mushroom, Eater, Toast])[0];
-    let monsterType = shuffle([Rock, Paper, Scissors ,Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti])[0];
+
+     let monsterType = shuffle(chooseMonsterType(level))[0];
```

Comment:
Look out for the identation here :ok_woman:

Reply:
And btw, why let and not const here?

Reply:
yeah, good call. I will change that.